docs/details/linux.md

```diff
@@ -0,0 +1,39 @@
+title: Zero Install for Linux
+
+The Linux version of Zero Install extends the cross-platform core [Zero Install .NET](../developers/dotnet-api.md) with a GUI and various OS-specific integrations.
+
+The Linux version and [Windows version](windows.md) of Zero Install share the same [feed format](../specifications/feed.md). For most common use-cases they behave identically on the command-line. There are however some [OS-specific differences](os-differences.md).
+
+[TOC]
+
+# Installation
+
+You can download download pre-compiled binaries and various distribution-specific packages [here](http://0install.net/install-linux.html).
+
+To use the pre-compiled binaries download the appropriate archive for your system, unpack it, and run the `install.sh` script inside. You'll need `libcurl3` installed (many systems have it by default). Use e.g.
+
+```shell
+apt-get install libcurl3
+```
+
+to get it on Ubuntu. 
+
+# From source
```

Comment:
This needs to explain that these instructions get the latest developer version. It should probably link to the GitHub releases page for stable releases too.

Reply:
I've changed the text to present downloading a source archive from GitHub releases and running `git clone` as alternatives.